frontend/src/app/libs/auth-pages/sign-in/sign-in.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { FormGroup, Validators, FormBuilder, AbstractControl } from '@angular/forms';
+import { UserCredentials } from '../../auth/models/user-credentials';
+import { AuthFacadeService } from '../../auth/store/auth/auth.facade';
+import { Router } from '@angular/router';
+import { ValidatorError } from '../models/validator-errors';
+import { EntityStatus } from '../../auth/models/entity-status';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { User } from '../../auth/models/user';
+import { VisibilityIcon } from '../models/visibility-icon';
+import { EntityWrapper } from '../../auth/models/entity-wraper';
+
+@Component({
+	selector: 'app-sign-in',
+	templateUrl: './sign-in.component.html',
+	styleUrls: ['./sign-in.component.scss'],
+})
+export class SignInComponent implements OnInit, OnDestroy {
+	private passwordMinLenth: number = 6;
+	private destroy: Subject<boolean> = new Subject<boolean>();
```

Comment:
Please follow rxjs name convention, observable variables should have postfix `$`, so `destroy` => `destroy$`

Reply:
done